README.md

```diff
@@ -1293,26 +1289,26 @@ The other advantage is that you don't have to write and run a program that does
 Here we are exporting the consumer API as public service with a stream as response (meaning more than one message as response).
 This can also be done as private export which requires a token signed by the exporting account for the importing account. Therefore you have precise control on who can import.
 You can also export the entire JetStream API by exporting `$JS.hub.API.>`.
-If you do so, you are giving full control over JS to everyone importing.
+If you do so, you are giving full control over JetStream to everyone importing.
 I also export `$JS.hub.API` instead of `$JS.API`. 
-This is so that I can pin access to a particular JS domain and not just use to the one where I connect to.
+This is so I can pin access to a particular JetStream domain and not just to the one I connect to.
 
 On import we change `$JS.hub.API` to `JS.test@hub.API`. 
 This is done to stay clear of the $JS prefix which may get additions as new features are added to JetStream.
 We give it a different prefix and subsequently specify that prefix if we want to talk to that particular JetStream. 
 Btw this import renaming feature is generally available. 
```

Comment:
I think this sentence needs to be clarified whereas, was renaming not available before and what does it relate to in the previous text as I don't see anything about renaming per se above.